### PR TITLE
Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by Amazinggracee**
### Learning objectives

- [ ] I gave feedback on someone else's project based on clean code best practices.

- [ ] I applied the knowledge gained in the past when giving a code review.

- [ ] I performed a simple code review for a coding partner.

- [ ] I understood existing code in a way that allows me to give a good code review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Click Here]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
```
Comment (addisalemseifu) [inline README.md:59]: - if you have already published your project it's better to include the link. and of not just include a text `not published`.

File: README.md [hunk 2]
```diff
@@ -1 +1,171 @@
...
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (addisalemseifu) [inline README.md:171]: - Please make sure the license is working.

**Review by addisalemseifu — CHANGES_REQUESTED** [review]
You have done a great job on your project but there are some improvements you have to make to proceed to the next step.
-Good readme file.
-Good commit message.
-Try to Use tenses like Added new features for the todo list.

**Comment by addisalemseifu** [comment]
### ****status :recycle:****


229514525-8e23ea47-0e1c-42d0-aaa7-9543c9dd5dee
> ### Learning objectives
> * [ ]  I gave feedback on someone else's project based on clean code best practices.
> * [ ]  I applied the knowledge gained in the past when giving a code review.
> * [ ]  I performed a simple code review for a coding partner.
> * [ ]  I understood existing code in a way that allows me to give a good code review.


OPTIONAL- After clearing all completed task, the index for the tasks do not reset, please fix this 🙏
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

File: README.md (hunk 1 above)
Comment (Amazinggracee) [reply README.md:59]: I have implemented your suggestions.

File: README.md (hunk 2 above)
Comment (Amazinggracee) [reply README.md:171]: Copy that.

**Comment by addisalemseifu** [comment]
> ### Learning objectives
> * [ ]  I gave feedback on someone else's project based on clean code best practices.
> * [ ]  I applied the knowledge gained in the past when giving a code review.
> * [ ]  I performed a simple code review for a coding partner.
> * [ ]  I understood 
229794846-64d98097-4d6a-4584-986a-98b6c0303f23
existing code in a way that allows me to give a good code review.

- you have successfully completed the project there is nothing left to do other than merge your project.

**Review by addisalemseifu — APPROVED** [review]

**Pull request merged**